src/plugins/intel_gpu/src/graph/include/layout_optimizer.h

```diff
@@ -196,7 +196,8 @@ class layout_optimizer {
     // Returns whether reorder between "prev" with format fmt_prev and "next" with format fmt_next
     // can be fused into next.
     bool can_fuse_reorder(program_node& prev, program_node& next, format fmt_prev, format fmt_next);
-    bool can_fuse_reorder_to_prev(program_node& prev, program_node* next, format fmt_prev, format fmt_next);
+    // Can the reorder 'target_node' be fused into prev
+    bool can_fuse_reorder_to_prev(program_node& prev, program_node& target_node, format fmt_prev, format fmt_next);
```

Comment:
If the target_node is supposed to be reorder, then change the type of argument to `reorder_node` instead of adding comment

Reply:
And the motivation of the change is unclear, as you still use next inside the implementation..

Reply:
There was a logic to refer to next.get_dependency(0) to check output format.
The target reorder of the case was located at index 1. So it used a wrong dependency.

Reply:
> If the target_node is supposed to be reorder, then change the type of argument to `reorder_node` instead of adding comment

Applied